src/components/organisms/navigation/stories/nav.stories.native.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { Text, View } from 'react-native';
+
+// import TabBar from '../nav.native';
+
+storiesOf('Tab Bar', module)
+  .addDecorator(storyFn => (
+    <View>
+      {storyFn()}
+      <View>
+        <Text style={{ textAlign: 'center' }}>Tab Bar Navigation</Text>
+      </View>
+    </View>
+  ))
+  // .add('with 3 items', () => <TabBar />);
```

Comment:
@AlanSl I could not figure out how to use react native navigation with storybook, please look if you have time.

Reply:
This is probably related to the issue I'm having with Jest. Drawer navigation seems to be buggy, so I'm looking into creating a navigator where the navigation logic is less tightly coupled to the UI.

When I've got it working I'll test it with Storybook and hopefully it'll solve four issues at once:

 - Fix universal tests
 - Better Native animations for screen changes
 - Easier to apply a different navigation UI without changing routing logic
 - Easier to use navigation-related components and hooks in Storybook